app/models/meeting.rb

```diff
@@ -21,7 +21,7 @@ class Meeting < ActiveRecord::Base
 
   has_many :members, -> { order 'name' }, through: :meeting_members,
                                           source: :user
-  has_many :meeting_members, foreign_key: :meetingid
+  has_many :meeting_members, foreign_key: :meetingid, dependent: :destroy
```

Comment:
The [relevant docs for `dependent: delete_all`](http://api.rubyonrails.org/v4.1.8/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many) state that:

> :delete_all causes all the associated objects to be deleted directly from the database (so callbacks will not be executed).

So in the event that callbacks are added to the `MeetingMember` model, changing this to `dependent: delete_all` will silently ignore them, which I imagine will be problematic. So unless `GroupsController#destroy` is shown to be a performance hotspot, I would suggest leaving as is.